testutils/README.md

```diff
@@ -0,0 +1,22 @@
+Baseline Checks
+===============
+Scripts in this directory sanity-checks the `*.baseline` files.
+
+check-baselines.sh
+------------------
+Checks that string `$unhandledCallingForm` does not appear in baseline files,
+since it is used as a placeholder for unimplemented generator features.
+
+Checks that every calling form defined in `CallingForm.java` is used by at least
+one baseline.
+
+From the root of the repository, run `./testutils/check-baselines.sh`
+
+check-calling-form-in-baseline
+------------------------------
+Checks that calling forms and value sets defined in GAPIC yaml file is used in baseline.
```

Comment:
s/defined/tagged/ ? wdyt?

"in the *_gapic.yaml files"
"in the baselines"

Reply:
Done